src/main/webapp/order.html

```diff
@@ -6,21 +6,43 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
   <body onload="determineLogin()" id= "background">
 
-<nav id="navbar">
-   <a href="index.html" class="button">Home</a>
+  <nav id="navbar">
+   <a href="index.html" id="active" class="button">Home</a>
    <a href="user.html" class="button">Profile</a>
-   <a href="store.html" class="button">Explore</a> 
+   <a href="store.html" class="button">Explore</a>
+   <a href="order.html" class="button">Order</a> 
```

Comment:
order tab should be the one labeled active here

Reply:
Didn't notice the mistake.